### PR TITLE
Add three options to warning ignore sub-directory and ignore.md

### PR DESCRIPTION
**Pull request by benhoyle**
Add three forks to one of the sub-stories.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2571)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/warning-ignore/ignore.md [hunk 1]
```diff
@@ -3,3 +3,8 @@ to a rustling sound. Deciding to investigate you go to the other room only
 to find it filled with rats. There is a moment of silence when the rats stop
 moving and stare at you. Then suddenly, as quickly as a flash of lightening
 the rats begin to swarm you.
+
+Do you:
+- [grab the nearest item to hand and throw it at the rats?](rat-fight/flaming_rats.md)
+- [turn and run out the room?](rat-fight/ice-berg.md)
```
Comment (casey-collab) [inline english/warning-ignore/ignore.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/warning-ignore/ignore.md [hunk 2]
```diff
@@ -3,3 +3,8 @@ to a rustling sound. Deciding to investigate you go to the other room only
...
+- [lie down?](rat-fight/sleep.md)
```
Comment (casey-collab) [inline english/warning-ignore/ignore.md]: This link is also on the line directly after another link.

File: english/warning-ignore/rat-fight/flaming_rats.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+You grab the nearest item to hand and throw it at the rats. The item
```
Comment (casey-collab) [inline english/warning-ignore/rat-fight/flaming_rats.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/warning-ignore/rat-fight/ice-berg.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+You turn and run out of the room. At that moment there is the loud noise
```
Comment (casey-collab) [inline english/warning-ignore/rat-fight/ice-berg.md]: This file has also been added to a directory that already contained a page.

File: english/warning-ignore/rat-fight/sleep.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+You lie down and the rats swarm all over you. You feel their hairy bodies,
```
Comment (casey-collab) [inline english/warning-ignore/rat-fight/sleep.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**